### ISSUE TITLE
[1.3] Fullscreen doesn't work

### ISSUE THREAD

**Issue by donizettilorenzo** [MadDeCoDeR/Classic-RBDOOM-3-BFG#109]
It seems the game can't retrieve fullscreen mode list (GeForce 3060).
No problem in windowed mode.

**Comment by MadDeCoDeR** [comment]
Hi, have you try to unplug the second monitor? Some monitors might still report that they are active even though they aren't, just because the monitor is still connected

**Comment by donizettilorenzo** [comment]
Just let me try (it's the same monitor, simply connected with another input port to the iGPU :p )

**Comment by donizettilorenzo** [comment]
Culprit was the second monitor :D
Solved!

https://i.postimg.cc/vTvhmgtc/Doom-BFA-2023-08-10-14-43-39-542.jpg

**Comment by cardcathouse** [comment]
I am experiencing this issue too, but I am using a laptop with dual GPUs (Nvidia/Intel). External monitor is connected to the Nvidia GPU through HDMI and laptop monitor is disabled. Is there a way to play using the external monitor only?

**Comment by MadDeCoDeR** [comment]
Have you try the launch argument +set r_fullscreen 2 ?

**Comment by cardcathouse** [comment]
Yes, but the game still opens in a small window. I can only choose 720p, borderless or windowed. I tried using +set r_customWidth 1920 +set r_customHeight 1080 but that didn't work either.

**Comment by MadDeCoDeR** [comment]
Can you try to run it with +set logfile 1 and post here the logfile?

**Comment by cardcathouse** [comment]
Sure, here's the logfile.
[qconsole.log](https://github.com/MadDeCoDeR/Classic-RBDOOM-3-BFG/files/12598865/qconsole.log)

**Comment by MadDeCoDeR** [comment]
Hi, sorry for the late response. Sadly the log file wasn't that helpfull on that issue (it was helpfull on another issue related to nVidia GPUs). So I decided to add some additional logging when the engine tries to retrieve monitor resolutions from the OS.
If any of you can please try to replicate the issue with the launch argument +set logfile 1 and send me the log file (here)

**Comment by cardcathouse** [comment]
Sure, I'll give it a try over the weekend. I'll let you know when I got the log file.

**Comment by cardcathouse** [comment]
[qconsole.log](https://github.com/MadDeCoDeR/Classic-RBDOOM-3-BFG/files/12776275/qconsole.log)
Here's the log file. I ran the game with the following arguments:
"+set r_fullscreen 2 +set logfile 1 +set r_customWidth 1920 +set r_customHeight 1080"
Please let me know if you need me to run the game with different arguments.

**Comment by MadDeCoDeR** [comment]
The message was empty. I added also the error code and try to force it to print it in english (using my evil eye). Can you try again with the newest nightly build?

P.S.: Sorry for the inconvinience

**Comment by cardcathouse** [comment]
[qconsole.log](https://github.com/MadDeCoDeR/Classic-RBDOOM-3-BFG/files/12777940/qconsole.log)
No problem, I'm happy to help as much as I can. Here's the log with the new nightly build. I ran the game with the same arguments as in my last comment.

**Comment by MadDeCoDeR** [comment]
So I removed yet another check up that the engine was doing when it was retrieving monitor resolutions. I hope that it works now. You can check the latest nightly build to verify

**Comment by cardcathouse** [comment]
Seems to work just fine! Thanks for all your work! I'll test it further when I have a little more time but all seems good so far.
